prow/config.yaml

```diff
@@ -6653,6 +6653,66 @@ periodics:
       - name: GOPATH
         value: /go
 
+- name: ci-containerd-build
+  interval: 30m
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180314-34ab1109a-experimental
+      args:
+      - "--repo=github.com/containerd/containerd=master"
+      - "--repo=github.com/containerd/cri=master"
+      - "--root=/go/src"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+- interval: 1h
+  agent: kubernetes
+  name: ci-containerd-e2e-gci-gce
```

Comment:
what's the difference betweeh this and `ci-cri-containerd-e2e-gci-gce`?

Reply:
This test is for `github.com/containerd/containerd` repository, not `github.com/containerd/cri`.